src/Go.CodeAnalysis/Common/Error.cs

```diff
@@ -2,6 +2,9 @@
 {
     using Go.CodeAnalysis.Text;
 
+    /// <summary>
+    /// Represents an item in Error List.
```

Comment:
Nit: this layer doesn’t know about the error list. This just represents an error in a span of text.